Formula/ib.rb

```diff
@@ -0,0 +1,65 @@
+class Ib < Formula
+  desc "C/C++ build tool that automatically resolves included source"
+  homepage "https://github.com/JasonL9000/ib"
+  url "https://github.com/JasonL9000/ib/archive/0.7.1.tar.gz"
+  sha256 "a5295f76ed887291b6bf09b6ad6e3832a39e28d17c13566889d5fcae8708d2ec"
+  depends_on :python
+
+  def install
+    prefix.install "common.cfg", "debug.cfg", "release.cfg", "asan.cfg", "__ib__"
+    bin.install "ib"
+  end
+
+  # assumes gcc is available
+
+  test do
+    example_config = <<-EOF.gsub(/^\s+/, "")
```

Comment:
Use `EOS.undent` instead of `EOF.gsub(/^\s+/, "")`. Also, use `(testpath/"example/debug.cfg").write` with this block instead of another call to write below.